### PR TITLE
[beta] Fix Chai recipe in docs

### PR DESCRIPTION
**Pull request by ashrene-roy**
Here's a simple and accurate Masala Chai recipe to update the one given here in the docs (https://beta.reactjs.org/learn/keeping-components-pure)

### CODE REVIEW

File: beta/src/content/learn/keeping-components-pure.md [hunk 1]
```diff
@@ -53,9 +53,12 @@ React is designed around this concept. **React assumes that every component you
 function Recipe({ drinkers }) {
   return (
     <ol>    
-      <li>Boil {drinkers} cups of milk.</li>
-      <li>Add {2 * drinkers} spoons of masala spices.</li>
-      <li>Remove from heat, and add {drinkers} spoons of tea.</li>
+      <li>Boil {drinkers} cups of water.</li>
```
[the remark this answers is not in the document]
Comment (ashrene-roy) [reply beta/src/content/learn/keeping-components-pure.md:56]: Let me try, with a little bit of chatGPT magic, it should be.

File: beta/src/content/learn/keeping-components-pure.md [hunk 2]
```diff
@@ -75,15 +78,15 @@ export default function App() {
...
 
-If you pass `drinkers={4}`, it will return JSX containing `4 cups of milk`. Always. 
+If you pass `drinkers={4}`, it will return JSX containing `4 cups of water`. Always. 
 
 Just like a math formula. 
 
 You could think of your components as recipes: if you follow them and don't introduce new ingredients during the cooking process, you will get the same dish every time. That "dish" is the JSX that the component serves to React to [render.](/learn/render-and-commit)
 
-<Illustration src="/images/docs/illustrations/i_puritea-recipe.png" alt="A tea recipe for x people: take x cups of water, add 2x spoons of spices, and x spoons of tea!" />
+<Illustration src="/images/docs/illustrations/i_puritea-recipe.png" alt="A tea recipe for x people: take x cups of water, add 0.5x spoons of spices, and x spoons of tea!" />
```
[the remark this answers is not in the document]
Comment (ashrene-roy) [reply beta/src/content/learn/keeping-components-pure.md:86]: Sure.

File: beta/src/content/learn/keeping-components-pure.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ashrene-roy) [reply beta/src/content/learn/keeping-components-pure.md:56]: Under 3 lines indeed. I'm able to see the second part in mobile view.

**Comment by gaearon** [comment]
awesome. thanks!

**Pull request merged**